.circleci/src/commands/restore_pods.yml

```diff
@@ -0,0 +1,17 @@
+description: Restores and saves the cocoa pods cache.
+steps:
+  - restore_cache:
+      name: Restore CocoaPods Cache
+      keys:
+        - 6-pods-{{ arch }}-{{ checksum "frontend/ios/Podfile.lock" }}
```

Comment:
same as above. i think its the wrong number

Reply:
It is the same, above is the auto generated config.yml